spinalcordtoolbox/scripts/sct_register_multimodal.py

```diff
@@ -215,9 +215,9 @@ def get_parser():
               f"  - pca_eigenratio_th: <int> Min ratio between the two eigenvalues for PCA-based angular adjustment "
               f"(only for algo=centermassrot and rot_method=pca). "
               f"Default={DEFAULT_PARAMREGMULTI.steps['1'].pca_eigenratio_th}.\n"
-              f"  - dof: <str> Degree of freedom for type=label. Separate with '_'. T stands for translation and R "
-              f"stands for rotation, x, y, and z indicating the direction. For example, Tx_Ty_Tz_Rx_Ry_Rz would allow "
-              f"translation on x, y and z axes and rotation on x, y and z axes. "
+              f"  - dof: <str> Degree of freedom for type=label. Separate with '_'. T stands for translation, R stands "
+              f"for rotation, and S stands for scaling. x, y, and z indicate the direction. For example, "
+              f"Tx_Ty_Tz_Rx_Ry_Rz would allow translation on x, y and z axes and rotation on x, y and z axes. "
```

Comment:
Due to the complexity of this argument, I think it might be helpful to add a second example:

```suggestion
              f"for rotation, and S stands for scaling. x, y, and z indicate the direction. For example:\n"
              f"  - 'Tx_Ty_Tz_Rx_Ry_Rz' would allow translation on x, y and z axes and rotation on x, y and z axes.\n"
              f"  - 'Tx_Rx_Sx' would perform translation, rotation, and scaling on just the x axis."
```

Reply:
The addition of a second example is a great idea! I [added](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3948/commits/2285efa7095fa3bd9793f488aa1bae50d530afe3) a bit more practical example (`Tx_Ty_Tz_Sz`) which is used, for example, within the `sct_register_to_template`. I also moved the default value to the beginning of the argument.